knative-operator/deploy/resources/kourier/kourier.yaml

```diff
@@ -0,0 +1,275 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kourier-system
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-logging
+  namespace: kourier-system
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-observability
+  namespace: kourier-system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: kourier
+  namespace: kourier-system
```

Comment:
I think we should deploy all kourier components (or only kourier gateway) in `knative-serving-ingress` namespace.
Otherwise, k-o-i needs to create OpenShift's Route in the same namespace with 3scale-kourier-gateway.

/cc @bbrowning

Reply:
I'm deploying the control plane and the gateway in the knative-serving-ingress namespace. Working fine in my cluster...

Reply:
I'm deploying the control plane and the gateway in the knative-serving-ingress namespace. Working fine in my cluster...